build.sbt

```diff
@@ -40,8 +40,8 @@ lazy val etcd = project
   .settings(
     name := "reactive-config-etcd",
     libraryDependencies ++= Seq(
-      "io.grpc"              % "grpc-netty"                      % "1.24.0",
-      "io.netty"             % "netty-tcnative-boringssl-static" % "2.0.7.Final",
+      "io.grpc"              % "grpc-netty"                      % "1.22.3",
```

Comment:
Downgrade?

Reply:
Actually I think we can add option `%  "provided"` here to use client version.